cdc/cdc/kv/region_worker.go

```diff
@@ -730,39 +731,45 @@ func (w *regionWorker) handleResolvedTs(
 		return nil
 	}
 	regionID := state.sri.verID.GetID()
-	// Send resolved ts update in non blocking way, since we can re-query real
-	// resolved ts from region state even if resolved ts update is discarded.
-	// NOTICE: We send any regionTsInfo to resolveLock thread to give us a chance to trigger resolveLock logic
-	// (1) if it is a fallback resolvedTs event, it will be discarded and accumulate penalty on the progress;
-	// (2) if it is a normal one, update rtsManager and check sinceLastResolvedTs
-	select {
-	case w.rtsUpdateCh <- &regionTsInfo{regionID: regionID, ts: newResolvedTsItem(resolvedTs)}:
-	default:
-	}
 
-	if resolvedTs < state.lastResolvedTs {
+	lastResolvedTs0, lastResolvedTs := state.lastResolvedTs[0], state.lastResolvedTs[1]
```

Comment:
Suggest to encapsulate the logics in `regionFeedState`, and return whether the incoming `resolvedTs` is normal.